### ISSUE TITLE
sync tmm queues

### ISSUE THREAD

**Issue by 1-alex98** [FAForever/server#924]
Sync timer of tmm queues and consider all possible games across queues

**Comment by 1-alex98** [comment]
@BlackYps I heared you are also doing something in that thing?

**Comment by BlackYps** [comment]
Yes, on the last board meeting we decided to slightly alter the matchmaker configuration and to reduce the pop timer.
Bit these are no complicated changes

**Comment by 1-alex98** [comment]
So Strife and are on this issue here
